presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -139,6 +144,39 @@ property
     : identifier EQ expression
     ;
 
+parameterDeclaration
+    : identifier type
+    ;
+
+routineCharacteristics
+    : routineCharacteristic*
+    ;
+
+routineCharacteristic
+    : language
+    | determinism
+    | sqlDataAccess
+    | nullCallClause
+    ;
+
+language
+    : SQL
```

Comment:
`LANGUAGE` is a keyword. It should be `LANGUAGE language_name`

Reply:
Updated `routineCharacteristic` rule